ndg/src/html/search.rs

```diff
@@ -18,9 +19,9 @@ use crate::{config::Config, html};
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SearchDocument {
   id:           String,
-  title:        String,
+  pub title:    String,
   content:      String,
-  path:         String,
+  pub path:     String,
```

Comment:
This is not very important but just fyi:

```rs
pub(crate) title: String,
pub(crate) path: String,
```

Should work for tests. Change if you care to.